### DISCUSSION TITLE
RocketMQ 支持生产者动态创建账号、密码？

### DISCUSSION THREAD

**Discussion by jinchunzhao** [apache/rocketmq#7067] [Show and tell]
开发语言：java
需求：生产者在发布指定topic消息时，动态创建用户（账号、密码），指定该用户才可以订阅接收该topic的消息。即订阅者订阅数据的权限控制

请问支持这样的操作？

**Comment by zhiliatom** [comment]
You can update plain_acl.yaml and create new ak/sk without restart servcie.

**Comment by onejimmyboy** [comment]
创建用户的话，第一个方法如楼上，第二个方法可以在java代码中使用DefaultMQAdminExt 创建用户密码。
